index.src.html

```diff
@@ -3624,6 +3646,11 @@
 
     When invoked, run the [=Close VideoFrame=] algorithm with [=this=].
 
+: <dfn method for=VideoFrame>getMetadata()</dfn>
+:: Gets the {{VideoFrameMetadata}} associated with this frame.
+
+    When invoked, return [=this=] {{VideoFrame/[[metadata]]}}.
```

Comment:
Per above, this may need to be a new copy each time, unless we deeply freeze the deserialized data.

Reply:
The idea is to copy the data at constructor time and deep freeze it if we can.
Otherwise, agreed we need to make a copy.

Reply:
I added a copy.